monai/transforms/croppad/array.py

```diff
@@ -214,6 +215,9 @@ def __call__(self, img: np.ndarray, mode: Optional[Union[NumpyPadMode, str]] = N
 class SpatialCrop(Transform):
     """
     General purpose cropper to produce sub-volume region of interest (ROI).
+    If a dimension of the expected ROI size is bigger than the input image size, will not crop that dimension.
+    So the cropped result may be smaller than expected ROI, and the cropped results of several images may not
+    have exactly same shape.
```

Comment:
... be smaller than **the** expected ROI, and the cropped results of several images may not have exactly **the** same shape.

Same for other docstring.

Reply:
Sure, updated in the latest commit.

Thanks.